.github/workflows/setup-fortran.yml

```
@@ -42,4 +42,4 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - run: |
           # Build with FPM
-          fpm build
+          fpm build --flag ${{ matrix.toolchain.FFLAGS }}
```

If the above suggestion is applied I don't think we need this here

```suggestion
          fpm build
```